test/obsidian/note_spec.lua

```diff
@@ -20,7 +20,7 @@ describe("Note", function()
     local note = Note.from_file "README.md"
     assert.equals(note.id, "README")
     assert.equals(#note.aliases, 1)
-    assert.equals(note.aliases[1], "Obsidian.nvim")
+    assert.equals(note.aliases[1], "obsidian.nvim")
```

Comment:
This seems like it might be unrelated and might be deserving of its own message in the commit log

Reply:
This was just because I changed the title of the README from "Obsidian.nvim" to "obsidian.nvim" to be consistent with the repo name.